examples/text-classification/run_glue.py

```diff
@@ -85,30 +132,21 @@ def main():
             f"Output directory ({training_args.output_dir}) already exists and is not empty. Use --overwrite_output_dir to overcome."
         )
 
-    # Setup logging
-    logging.basicConfig(
-        format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
-        datefmt="%m/%d/%Y %H:%M:%S",
-        level=logging.INFO if training_args.local_rank in [-1, 0] else logging.WARN,
-    )
+    # Log on each process the small summary:
     logger.warning(
-        "Process rank: %s, device: %s, n_gpu: %s, distributed training: %s, 16-bits training: %s",
-        training_args.local_rank,
-        training_args.device,
-        training_args.n_gpu,
-        bool(training_args.local_rank != -1),
-        training_args.fp16,
+        f"Process rank: {training_args.local_rank}, device: {training_args.device}, n_gpu: {training_args.n_gpu}"
+        + f"distributed training: {bool(training_args.local_rank != -1)}, 16-bits training: {training_args.fp16}"
     )
-    logger.info("Training/evaluation parameters %s", training_args)
+    # Set the verbosity to info on main process only:
+    if is_main_process(training_args.local_rank):
+        logging.set_verbosity_info()
+    logger.info(f"Training/evaluation parameters {training_args}")
```

Comment:
Have you confirmed this actually works? It seems to me that you're setting the default verbosity level of the *root* logger (so the loggers of `transformers` and every file contained in it), but the logger of the current file isn't a child of this logger (it's in `examples/`, not in `src/transformers/`)so it doesn't look like it'll be impacted by that change.

I would argue you would still need to change the current logger's default verbosity to `info` if you want to see the line `logger.info(f"Training/evaluation parameters {training_args}")` being printed.

Reply:
It does work and I can see all the info being printed on my screen.

Reply:
Ok, this should be working now.